modules/sdk-test/package.json

```diff
@@ -33,7 +33,6 @@
     "@bitgo/statics": "^6.17.0",
     "bignumber.js": "^8.0.1",
     "lodash": "^4.17.14",
-    "should": "^13.1.3",
     "should-http": "^0.1.1"
```

Comment:
nit: do we need `should-http` if we don't have `should`?

Reply:
`should` is provided by the root package.